packages/lesswrong/server/scripts/importEAGUserInterests.ts

```diff
@@ -33,3 +36,68 @@ Vulcan.importEAGUserInterests = wrapVulcanAsyncScript(
       })
   }
 )
+
+
+type UserEAGDetailsRow = {
+  email: string,
+  careerStage?: string,
+  countryOrRegion?: string,
+  nearestCity?: string,
+  willingnessToRelocateBoston?: string,
+  willingnessToRelocateBayArea?: string,
+  willingnessToRelocateDC?: string,
+  willingnessToRelocateLondon?: string,
+  willingnessToRelocateNYC?: string,
+  willingnessToRelocateOxford?: string,
+  willingnessToRelocateRemote?: string,
+  experiencedIn?: string,
+  interestedIn?: string
+}
+
+/**
+ * Given a CSV with users' EAG data, import it to the "UserEAGDetails" table, replacing any existing user data.
+ */
+Vulcan.importEAGUserDetails = wrapVulcanAsyncScript(
```

Comment:
I have a couple `AnyBecauseTodo`s left in here - happy to take suggestions, otherwise I might give up since this is just a one-off script